packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx

```diff
@@ -84,6 +84,14 @@ export type CallContentProps = Pick<
      * If true, disables the Picture-in-Picture mode for iOS and Android
      */
     disablePictureInPicture?: boolean;
+    /**
+     * Props to set the audio mode for the InCallManager.
+     * If media type is video, audio is routed by default to speaker, otherwise it is routed to earpiece.
+     * Changing the mode on the fly is not supported.
+     * Manually invoke `InCallManager.start({ media })` to achieve this.
+     * @default 'video'
+     */
+    inCallManagerAudioMode?: 'video' | 'audio';
```

Comment:
I'd rename this prop to something else. Something like `defaultInCallManagerAudioMode` or `initialInCallManagerAudioMode` to signify that it works only during the initial render. Otherwise, customers might try changing it at runtime and not get the desired behavior.

Reply:
good point, updated the name to initialInCallManagerAudioMode